puppetry/kinectv2/pygame_button.py

```diff
@@ -0,0 +1,167 @@
+#!/usr/bin/env python3
+"""\
+@file pygame_button.py
+
+@brief Simple button class so this can have basic UI
+
+$LicenseInfo:firstyear=2022&license=viewerlgpl$
+Second Life Viewer Source Code
+Copyright (C) 2022, Linden Research, Inc.
+ 
+This library is free software; you can redistribute it and/or
+modify it under the terms of the GNU Lesser General Public
+License as published by the Free Software Foundation;
+version 2.1 of the License only.
+
+This library is distributed in the hope that it will be useful,
+but WITHOUT ANY WARRANTY; without even the implied warranty of
+MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the GNU
+Lesser General Public License for more details.
+
+You should have received a copy of the GNU Lesser General Public
+License along with this library; if not, write to the Free Software
+Foundation, Inc., 51 Franklin Street, Fifth Floor, Boston, MA  02110-1301  USA
+
+Linden Research, Inc., 945 Battery Street, San Francisco, CA  94111  USA
+$/LicenseInfo$
+"""

```

Comment:
This license header is no longer needed: https://wiki.secondlife.com/wiki/Coding_standard#MIT_Licensed_Code

Reply:
done, replaces with the newer license

Reply:
... next push will remove them